src/app/public/src/modules/restricted-view/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './restricted-view.directive';
+export * from './restricted-view.component';
+export * from './restricted-view.module';
```

Comment:
Barrel files within modules can cause some circular dependency issues.  Can we remove all barrels other than the base `index.ts` file and the `src/app/public/index.ts` files?